mobile/lib/features/wallet/create_invoice_screen.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:get_10101/common/amount_text_input_form_field.dart';
+import 'package:get_10101/common/domain/model.dart';
+import 'package:get_10101/features/wallet/domain/wallet_info.dart';
+import 'package:get_10101/features/wallet/share_invoice_screen.dart';
+import 'package:get_10101/features/wallet/wallet_change_notifier.dart';
+import 'package:get_10101/features/wallet/wallet_screen.dart';
+import 'package:go_router/go_router.dart';
+import 'package:provider/provider.dart';
+
+import 'application/wallet_service.dart';
+
+class CreateInvoiceScreen extends StatefulWidget {
+  static const route = "${WalletScreen.route}/$subRouteName";
+  static const subRouteName = "create_invoice";
+  final WalletService walletService;
+
+  const CreateInvoiceScreen({super.key, this.walletService = const WalletService()});
+
+  @override
+  State<CreateInvoiceScreen> createState() => _CreateInvoiceScreenState();
+}
+
+class _CreateInvoiceScreenState extends State<CreateInvoiceScreen> {
+  Amount? amount;
+  final TextEditingController _amountController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    WalletInfo info = context.watch<WalletChangeNotifier>().walletInfo;
+    log("Refresh receive screen: ${info.balances.onChain}");
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Receive funds")),
+      body: SafeArea(
+        child: Container(
+          constraints: const BoxConstraints.expand(),
+          child: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
+            const Center(
+              child: Padding(
+                padding: EdgeInsets.only(top: 25.0),
+                child: Text(
+                  "Share payment request",
```

Comment:
🔧 Please align with the figma wireframe.

"What is the amount?"

Reply:
Oops, I missed this :sweat_smile: